db/generateSeedData.js

```diff
@@ -143,59 +140,63 @@ var generateRelatedItems = (num) => {
 
         } else {
             for (var i = 0; i < rangeOfItems.length; i++) {
-                returnArr.push([rangeOfItems[i]]);
+                // returnArr.push([rangeOfItems[i]]);
                 rangeOfItems.splice(i, 1);
             }
         }
         
     }
-    console.log(returnArr);
+    // console.log(returnArr);
     return returnArr;
 };
 
-var generateFeatureRatings = (num) => {
+// Generate up to 4 random features for num items (this has an id attached to it and I am not sure how to deal with this)
+let generateFeatureRatings = (num) => {
     var featureRating = [];
     for (var i = 0; i < num; i++) {
         var featureObject = {
             typeOfFeature: faker.commerce.productAdjective(),
-            rating: Math.floor(Math.random() * (5-1)+1),
-            rating_id: i
+            rating: Math.floor(Math.random() * (4)),
+            id_reviews: i
```

Comment:
I think feature ratings are aggregated from all reviews for a given product, so you could consider using the product id instead?